Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
+        @version = OS::Mac::Version.new version
```

Comment:
You're using `OS::Mac` in this file, but `MacOS` everywhere else. I think we should be using the former for all new code.